src/strategies/rootroop-staking/index.ts

```diff
@@ -0,0 +1,46 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import abi from './xRooStakingABI.json';
+
+export const author = 'animalmix55';
+export const version = '0.0.1';
+
+interface UserData {
+  stake: BigNumberish;
+  liquidity: BigNumberish;
+  lastTimestamp: BigNumberish;
+  RTRewardModifier: BigNumberish;
+  NFTXRewardModifier: BigNumberish;
+  NFTXRewardWithdrawn: BigNumberish;
+}
+
+interface Params {
+  address: string;
+  decimals: number;
+  symbol: string;
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options: Params,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'users', [address])
+  );
+  const result: Record<string, UserData> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, userData]) => [
+      address,
+      parseFloat(formatUnits(userData.stake, options.decimals))
```

Comment:
Hi @Animalmix55 I think you can use https://snapshot.org/#/strategy/contract-call strategy for this

Reply:
I saw this strategy but I didn’t think it could handle a method that returns a strict rather than a uint. Am I incorrect?

Reply:
Hello again @Animalmix55 Nope, you can pass `"output":"stake"` param in strategy paramss